tests/run.sh

```diff
@@ -25,6 +25,6 @@ cleanup
 echo 'starting test run, tests will run in parallel and output shown at the end'
 
 export PYTHONPATH=$PWD/..:$PWD/../lib/
-nosetests --verbosity=3 --processes=10 --process-timeout=3600 --with-timer --timer-no-color usecase/test_*.py
```

Comment:
How come we don't want the verbosity or the timers?

Reply:
1) Output got truncated anyway
2) From the conversations we had, It was clear no one checks or cares about the execution time and/or verbose logs.
3) We got rid of the dependency

Reply:
Let's try to get timing back now test run okay, having a overview of all tests run if useful.